.ci-support/merge-install-production-branch.sh

```diff
@@ -12,28 +12,67 @@ set -x
 #
 # PRODUCTION_BRANCH = The production branch (default=production)
 # PRODUCTION_FORK = The production fork (default=illinois-ceesd)
-#
-MIRGE_HOME=${1:-"."}
-PRODUCTION_BRANCH=${PRODUCTION_BRANCH:-"production"}
-PRODUCTION_FORK=${PRODUCTION_FORK:-"illinois-ceesd"}
 
+MIRGE_HOME=${1:-"."}
 echo "MIRGE_HOME=${MIRGE_HOME}"
+
+if [[ "$GITHUB_HEAD_REF" == "$PRODUCTION_PR_MAIN_BRANCH" ]]; then
+  # GITHUB_HEAD_REF is only set for PR builds.
+  # https://docs.github.com/en/actions/learn-github-actions/environment-variables
+  PRODUCTION_BRANCH="production-merge/${PRODUCTION_PR_MAIN_BRANCH}"
+  PRODUCTION_FORK=${PRODUCTION_PR_FORK:-"illinois-ceesd"}
+else
+  PRODUCTION_BRANCH="${PRODUCTION_BRANCH:-"production"}"
+  PRODUCTION_FORK=${PRODUCTION_FORK:-"illinois-ceesd"}
+fi
```

Comment:
I think the `else` is unneeded.  Use what was set just above or default to production proper.

```suggestion
PRODUCTION_BRANCH="${PRODUCTION_BRANCH:-"production"}"
PRODUCTION_FORK=${PRODUCTION_FORK:-"illinois-ceesd"}
```

Reply:
I'm not sure I understand what you're asking for here. Your first comment seems to aim at removing `PRODUCTION_{FORK,BRANCH}`, and your second one seems to push for reinstating them. Which of the two would you like? And if you're interested in reinstating them, for what use case that isn't a PR?

Reply:
This change set forces a custom production branch called: `production-merge/${PR_MAIN_BRANCH}`.   I am accustomed to calling my custom production branch whatever I want:
`PRODUCTION_BRANCH=my-custom-production-branch`.    

I would like to retain that capability.   Nothing in this change set prevents my continuing to do that, except the comment snippet that indicates something like that is possible has been removed from `production_test_env.sh`.  I am merely suggesting that snippet should be restored.

Reply:
> I am merely suggesting that snippet should be restored.

If it's restored, wouldn't that confuse new users who might see two ways to do the same thing and be confused on which of them they should use? (To my mind, the one with the naming convention is preferable, so I tried to make that the path of least resistance.)

Reply:
To me it isn't the same thing.  It simply allows a user to specify outright the name of the custom production branch to use at their option.

With the new way you are introducing, one is forced to have an independent custom production for each branch/PR, with the advantage that it does not need to be "undone" before merge.   In the way that I typically operate, I can and do use a single custom production branch for several developments/branches/PRs.   I would like to retain that ability.